lib/presentation/pages/trip_panel_page/widgets/rounded_icon.dart

```diff
@@ -0,0 +1,26 @@
+import 'package:flutter/material.dart';
+import 'package:trip_tally/presentation/theme/app_colors.dart';
+import 'package:trip_tally/presentation/theme/app_dimensions.dart';
+import 'package:trip_tally/presentation/widgets/m3_widgets/svg_asset.dart';
+
+class RoundedIcon extends StatelessWidget {
```

Comment:
RoundedIcon it's too generic, I would name it TransportTypeCircleAvatar and make it accept only transport type not iconPath

Reply:
When you do that you can replace it also here:
`expense_item.dart`